src/dash-table/derived/edges/data.ts

```diff
@@ -30,17 +34,57 @@ export default memoizeOneFactory((
     traverse2(
         data,
         columns,
-        (datum, column, i, j) => edges.setEdges(i, j, getDataCellEdges(datum, i + offset.rows, column)(styles))
+        (datum, column, i, j) => edges.setEdges(i, j, getDataCellEdges(
+            datum,
+            i + offset.rows,
+            column,
+            false,
+            false
```

Comment:
Since border calculations are now done on active cells too - need a partial calculation that assumes active=False, selected=False for all cells